doc/command_ref.rst

```diff
@@ -174,22 +174,22 @@ Options
     Disable specific repositories by an id or a glob. This option is mutually exclusive with ``--repo``.
 
 ``--downloaddir=<path>``
-    Redirect downloaded packages to provided directory. The option has to by used together with \-\
-    :ref:`-downloadonly <downloadonly-label>` command line option or with
-    ``download`` command (dnf-plugins-core) or with ``system-upgrade`` command
+    Redirect downloaded packages to provided directory. The option has to be used together with the \-\
+    :ref:`-downloadonly <downloadonly-label>` command line option, with the
```

Comment:
Not sure about comma here

Reply:
This is one of those I'm actually quite sure is correct. It separates the options, if I replace the actual contents to make it short and clear, the sentance says: "The option has to be used with A, with B or with C." I don't think you can leave out the comma here.